### DISCUSSION TITLE
[SOLO PROGRAMACIÓN] Capturas de pantalla

### DISCUSSION THREAD

**Discussion by yahirsaavedra** [createchsoftware/armony#4] [Anuncios]
Con el fin de optimizar el ámbito de trabajo de Linux que se utilizará para todo el desarrollo del proyecto Armony, es necesario saber las especificaciones de la laptop de todos.

Lo único que tienes que hacer es publicar dos capturas de pantalla por este medio

**Primera captura**
Esta primera captura se consigue entrando a la configuración del sistema y haciendo clic en Sistema > Acerca de, o buscando en el menú Inicio "Acerca de mi PC".

Deben verse ciertos detalles como:
- Procesador
- Memoria RAM
- Tipo de sistema, si es 64 bits o no

**Segunda captura**
Para esta segunda captura debes dirigirte al Explorador de archivos y buscar "Este equipo".

Deben verse los discos duros que están insertados en el sistema y cuánto espacio usado y libre tienen cada uno.

**Vencimiento de esta tarea**
Tienen hasta el sábado 16 a las 11 PM para entregarla. Cualquier entrega tardía será registrada en la bitácora.

**Comment by Julian-Sandoval-x** [comment]
![este equipo](https://github.com/createchsoftware/armony/assets/76192811/47ceeec1-fe88-4c29-b1ab-f82dd9b8e76c)
![sistema](https://github.com/createchsoftware/armony/assets/76192811/095b4bf7-4e86-4d1e-a553-a620d5bd40f1)

**Comment by KrisG012** [comment]
![Screenshot 2024-03-15 123536](https://github.com/createchsoftware/armony/assets/159211014/6ecd7eb7-cc09-4558-bde0-fdd76634b2c4)
![Screenshot 2024-03-15 123406](https://github.com/createchsoftware/armony/assets/159211014/86e5805e-cbdf-4dc9-b28a-3fff0c90b875)

**Comment by NICOLExALV** [comment]
![image](https://github.com/createchsoftware/armony/assets/159110785/21c62e8a-5cf3-4a2a-a754-f4d065c8ee2a)
![image](https://github.com/createchsoftware/armony/assets/159110785/199d3ce4-2f7f-4385-b6cf-0832ec5c70ea)

**Comment by JorgeInge** [comment]
![image](https://github.com/createchsoftware/armony/assets/157444533/afa515b0-3a99-4ce3-a0e3-64437ccebef8)
![image](https://github.com/createchsoftware/armony/assets/157444533/5f231bec-c0f4-4e4f-91da-c30635a1e6be)

**Comment by yahirsaavedra** [comment]
![Captura de pantalla (247)](https://github.com/createchsoftware/armony/assets/51274659/5272bc6a-7131-4eb2-87f0-2b61d198a085)
![Captura de pantalla (248)](https://github.com/createchsoftware/armony/assets/51274659/5e6da046-6801-4a3f-adb7-d12893c27dc4)

**Comment by adriandrs** [comment]
![image](https://github.com/createchsoftware/armony/assets/56745155/292ad829-0cd5-459d-8ada-3173203e78b6)
![image](https://github.com/createchsoftware/armony/assets/56745155/b53e7d70-36fc-4a06-a252-6c01183572b2)

**Comment by yahirsaavedra** [comment]
Concluye el periodo de entregas. A partir de este punto se consideran entregas tardías.

**Comment by yahirsaavedra** [comment]
Se cierra periodo de entregas tardías. Se registraron 6 entregas y 2 personas que no entregaron.
